03-TimeConversionApp/TimeConverter/TimeConverter/ContentView.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  ContentView.swift
+//  TimeConverter App
+//
+//  Created by Phil on 09/02/2023.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+  
+  @State private var inputValue = 60.0
+  @FocusState private var inputFocus
+  @State private var outputValue = 0.0
+  @State private var inputUnit: TimeUnit = .seconds
+  @State private var outputUnit: TimeUnit = .minutes
+  
+  private enum TimeUnit: String, CaseIterable{
+    case seconds, minutes, hours, days
+    
+    var displayName: String {
+      rawValue.capitalized
+    }
+  }
+  
+  private let options = TimeUnit.allCases
+  
+  // Convert to a base unit
+  private var getSeconds: Double {
```

Comment:
You could get even more swifty by adding another property to the enum. Say you have something like this in the enum: 
```
var secondsMultiplier: Double {
	switch self {
	case .seconds: return 1
	case .minutes: return 60
	case .hours: return 60 * 60
	case .days: return 60 * 60 * 24
	}
}
```

You could then remove this property completely and change the `convertedValue` property to: 
```
private var convertedValue: String {
	let converted = (inputValue * inputUnit.secondsMultiplier) / outputUnit.secondsMultiplier
	return converted.formatted()
}
```

The nice thing about this approach is that it means you only define the multipliers in one place